### ISSUE TITLE
HTMLElement.shadowRoot is deprecated.

### ISSUE THREAD

**Issue by steventango** [puranjayjain/chrome-color-picker#19]
The contents of `atom-text-editor` elements are no longer encapsulated
within a shadow DOM boundary. Please, stop using `shadowRoot` and access
the editor contents directly instead.
```
HTMLElement.shadowRoot (C:\Users\Steven\AppData\Local\atom\app-1.19.0-beta5\resources\app\src\text-editor-element.js:13:10)
Object.toggle (C:\Users\Steven\.atom\packages\chrome-color-picker\lib\chrome-color-picker.coffee:244:42)
HTMLElement.subscriptions.add.atom.commands.add.chrome-color-picker:toggle (C:\Users\Steven\.atom\packages\chrome-color-picker\lib\chrome-color-picker.coffee:117:26)
CommandRegistry.handleCommandEvent (<embedded>:21817:35)
handleCommandEvent (<embedded>:21555:65)
CommandRegistry.dispatch (<embedded>:21718:25)
```

**Comment by puranjayjain** [comment]
duplicate of #15

**Comment by brndto** [comment]
Any movement on this? I use this extension frequently.

**Comment by puranjayjain** [comment]
Hi @branditodesign  I'll look into this

**Comment by thehumangerm** [comment]
still an issue

**Comment by xxyyzz2050** [comment]
please fix

**Comment by VishwaJay** [comment]
Still an issue.

**Comment by OldNick** [comment]
Still an issue
It's a pity as it's the only decent colour picker

Maybe someone should fork it and sort it out... unfortunately I lack the expertise

**Comment by rrroyal** [comment]
Fixed in #24. @puranjayjain please take a look at this one.

**Comment by puranjayjain** [comment]
@rrroyal merged will release it with a tag to apm
